exporter/exporterhelper/queued_retry.go

```diff
@@ -66,7 +74,8 @@ func DefaultQueueSettings() QueueSettings {
 		// This is a pretty decent value for production.
 		// User should calculate this from the perspective of how many seconds to buffer in case of a backend outage,
 		// multiply that by the number of requests per seconds.
-		QueueSize: 5000,
+		QueueSize:  5000,
```

Comment:
I was thinking about another setting for persistent data (since the number would be probably much different) but then maybe we should just keep the same setting to keep it simple